cloudigrade/util/tests/helper.py

```diff
@@ -92,3 +93,23 @@ def generate_dummy_describe_instance(instance_id=None, image_id=None,
         'SubnetId': subnet_id,
     }
     return mock_instance
+
+
+def generate_mock_sqs_message(id, body, receipt_handle):
```

Comment:
Avoid redefining the Python builtin `id`. Let's call this `message_id`.

Reply:
🤢 what have I done!?!?